packages/2018-transportation-systems/src/components/DeclineInRidership/index.js

```diff
@@ -1,25 +1,93 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { css } from 'emotion';
 
-import { CivicStoryCard, Placeholder } from '@hackoregon/component-library';
+import { CivicStoryCard, LineChart } from '@hackoregon/component-library';
+
+import { fetchRidershipOverTime } from '../../state/decline-in-ridership/actions';
+import {
+  isRidershipOverTimePending,
+  getRidershipOverTimeData,
+} from '../../state/decline-in-ridership/selectors';
+
+const sampleSimpleData = [
+  { x: 0, y: 20 },
+  { x: 10, y: 30 },
+  { x: 20, y: 50 },
+  { x: 30, y: 40 },
+];
+
+const cardLoading = css`
+  width: 100%;
+  padding: 50px;
+  text-align: center;
+  background: #EEE;
+`;
+
+const cardError = css`
+  width: 100%;
+  padding: 50px;
+  text-align: center;
+  background: #FDD;
+  border: 1px solid #C99;
+`;
 
 export class DeclineInRidership extends React.Component {
   componentDidMount() {
-    // initialize data here
+    this.props.init();
   }
 
   render() {
+    const {
+      isLoading,
+      ridershipOverTime,
+    } = this.props;
+
+    if (isLoading) {
+      return <div className={cardLoading}>Loading...</div>;
+    } else if (!ridershipOverTime) {
+      return <div className={cardError}>Could not render Plateau in Ridership</div>;
+    }
+
     return (
       <CivicStoryCard
-        title="Decline in Ridership"
+        title="Plateau in Ridership"
         slug="decline-in-ridership"
```

Comment:
We'll probably want to update the slug for this code too. Which needs to be done here and in `registry.js` in the same package.

Reply:
👍